### PR TITLE
adds version_check healthcheck

### PR DESCRIPTION
**Pull request by whereismyjetpack**
First stab at an version endpoint. addresses #828

### CODE REVIEW

File: lib/healthchecks/version_check.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module HealthChecks
+  class VersionCheck < OkComputer::Check
+    def check
+      version = ENV.fetch('APP_VERSION', nil)
```
Comment (awead) [inline lib/healthchecks/version_check.rb]: What about `version = ENV.fetch('APP_VERSION', 'unknown')` and that way there's always something to display? Or is it important that a missing version key be communicated as a failure?

**Review by awead — CHANGES_REQUESTED** [review]
Looks good! One idea, but we can dismiss it if you like.

File: lib/healthchecks/version_check.rb (hunk 1 above)
Comment (whereismyjetpack) [reply lib/healthchecks/version_check.rb]: I don't think it's important. "unknown" sounds good to me!

**Comment by whereismyjetpack** [comment]
I think this is ready to rock and roll.

**Review by awead — APPROVED** [review]

**Pull request merged**